modules/consensus/synchronize_test.go

```diff
@@ -438,12 +438,12 @@ func TestIntegrationRPCSendBlocks(t *testing.T) {
 		// Create the "remote" peer.
 		remoteCST, err := blankConsensusSetTester(filepath.Join(t.Name()+" - remote", strconv.Itoa(i)))
 		if err != nil {
-			t.Fatal(err)
+			t.Fatalf("%v: %s", tt.msg, err)
```

Comment:
I think `%s` doesn't work for the `error` type. We typically use `%v` everywhere unless we need specific formatting, e.g. specifying the precision of a float